ui/components/app/signature-requests-common-header/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './signature-requests-common-header';
```

Comment:
This can be named `SignatureRequestHeader`, word `common` is not needed I think.